packages/grid/_modules_/grid/components/menu/columnMenu/ColumnsMenuItem.tsx

```diff
@@ -0,0 +1,26 @@
+import MenuItem from '@material-ui/core/MenuItem';
+import * as React from 'react';
```

Comment:
Should we keep on this convention?

```suggestion
import * as React from 'react';
import MenuItem from '@material-ui/core/MenuItem';
```